sw/device/silicon_creator/rom_ext/rom_ext_common.ld

```diff
@@ -17,21 +17,42 @@ GROUP(-lgcc)
  */
 __DYNAMIC = 0;
 
+INCLUDE hw/top_earlgrey/sw/autogen/top_earlgrey_memory.ld
+
+MEMORY {
+  eflash_rom_ext(rx) : ORIGIN = 0x90000000, LENGTH = 0x80000 
+}
+
```

Comment:
I'm not sure this is the best approach.
Maybe move it to the `top_earlgrey_memory.ld`?

Reply:
Yeah that's a tricky one. I think leave it here for now given that it isn't a physical limitation, but we might want to move it to a header file later.

Reply:
I don't have a strong opnion on that, maybe the `eflash_virtual` would be reserved for the OS or the baremetal application. 
Anyway I'm happy to use it or move the `eflash_rom_ext` declaration to the `top_earlgrey_memory.ld`.